vue-frontend/src/components/jobs/PerksAndBenifits.vue

```diff
@@ -201,6 +201,14 @@ export default {
   flex-direction: column;
 }
 
+.flex-div div > img {
+  transform: scale(1.1);
```

Comment:
The scale should be 1 here,  bcoz we have to make changes only on hover, so we can increase or decrease the scale on hover only.